manifests/r/ranpha/LavFiltersMegamix/0.77.1-2/ranpha.LavFiltersMegamix.locale.en-US.yaml

```diff
@@ -0,0 +1,25 @@
+PackageIdentifier: ranpha.LavFiltersMegamix
+PackageVersion: 0.77.1-2
+PackageLocale: en-US
+Publisher: Hendrik Leppkes
+PackageUrl: https://anime.my/tutorials/watching-h264-videos-using-compute-unified-device-architecture-cuda/
+PublisherUrl: https://anime.my
+PublisherSupportUrl: https://anime.my/misc/contact-form/
+PrivacyUrl: https://anime.my/misc/privacy-policy/
+Author: ranpha
+PackageName: LAV Filters 0.77.1-2
+License: Freeware
+ShortDescription: LAV Filters Megamix is a codec pack of LAVFilters, ASSFilterMod and Madvr with video players like PotPlayer and MPC-HC.
```

Comment:
```suggestion
ShortDescription: LAV Filters Megamix is a multiple codec pack with several popular video players included.
```

Reply:
This suggestion would clear the label `Policy-Test-1.2` by removing other product names.